src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,91 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
```

Comment:
This is how your tests should be named:
register_addAgeGreaterThanMinAge_ok
register_addAdultUser_ok
register_addInvalidUsersAge_NotOk
register_addNullAge_NotOk
register_addNullUser_NotOk
register_addNullLogin_NotOk
register_loginExists_NotOk
register_passwordLengthIsGreaterThanMinLength_Ok
register_passwordLengthMinLength_Ok
register_addNullPassword_NotOk
register_passwordLengthIsLessThanMinLength_Ok